Framework7/test_workspace_get.rb

```diff
@@ -0,0 +1,5 @@
+require_relative '../Framework7/src/helpers/rest_client/api_rest_client'
```

Comment:
This file should not be necessary since that it will not be executed during the execution of cucumber. you could create this file only for your development test